deployability/plugins/influxdb_reporter/setup.py

```diff
@@ -0,0 +1,38 @@
+from os import path
+
+from setuptools import setup
+
+
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
+setup(
+    name='pytest-influxdb',
+    description='A pytest plugin to report test results to influxdb',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    packages=['pytest_influxdb'],
+    author='quebim',
+    author_email='kevinledesmam95@gmail.com',
```

Comment:
leaks leaks leaks

Please remove sensitive data

Reply:
Done in [6311d92](https://github.com/wazuh/wazuh-qa/pull/5190/commits/6311d92724994e505494d41ca00dd7d9cddda76f)